LiskHQ/lisk-sdk
# Add initial dashboard plugin package - Closes #6218, #6219, #6221, #6222

### Description

- Add basic structure of the plugin
- Add Text component for sample
- Add Redux setup
- Add CSS constants
- Add empty main page

### What was the problem?

This PR resolves #6218, resolves #6219, resolves #6221, resolves #6222

### How was it solved?
- Copied most of the structure from the faucet plugin
- Add CSS constants according to the design
- Add `Text` component as a sample

### How was it tested?

- Run `npm run start:web` and see the design
- Check folder structure
